File: src/main/java/com/springMvcPractice/domain/model/Answer.java
package com.springMvcPractice.domain.model;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
@Getter
public class Answer {
    private final Drill drill;
    private final Integer calcResult;
    private Boolean judge;

    public void setJudge() {
        this.judge = this.calcResult.equals(drill.getAnswerInput());
    }
}

File: src/main/java/com/springMvcPractice/domain/model/Answer.kt
package com.springMvcPractice.domain.model

import lombok.Getter
import lombok.RequiredArgsConstructor

@RequiredArgsConstructor
@Getter
class Answer(val drill: Drill, val calcResult: Int) {
    var judge: Boolean = false
    fun setJudge() {
        judge = calcResult == drill!!.answerInput
    }
}
